### PR TITLE
Added plotting tool to visualize changes.

### PR DESCRIPTION
**Pull request by fpernice-google**
Script to plot a comparison between the data downloaded by script in #192  and simplified in #194 .

### CODE REVIEW

File: tools/geojson_simplifier/plotter.py [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+FLAGS = flags.FLAGS
+flags.DEFINE_string('original_path', default=None,
+                    help='Path to original geojson to be compared.')
+flags.DEFINE_string('simplified_path', default=None,
+                    help='Path to simplified geojson to be compared.')
+
+
+def main(_):
+    flags.mark_flag_as_required('original_path')
+    flags.mark_flag_as_required('simplified_path')
```
Comment (jeffreyoldham) [inline tools/geojson_simplifier/plotter.py:39]: Let's move these immediately after the flags declarations.

File: tools/geojson_simplifier/plotter.py [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+    python3 plotter.py --original_path original-data/geoId-01.geojson
+                       --simplified_path simplified-data/geoId-01-simple.geojson
+"""
+
+from absl import flags
+from absl import app
+import geopandas as gpd
+import matplotlib
+matplotlib.use('TkAgg')
+import matplotlib.pyplot as plt
```
Comment (jeffreyoldham) [inline tools/geojson_simplifier/plotter.py:27]: Do we need to update the requirements.txt file?

Comment (fpernice-google) [reply tools/geojson_simplifier/plotter.py:27]: Nope. These packages are already included.

File: tools/geojson_simplifier/plotter.py (hunk 1 above)
Comment (fpernice-google) [reply tools/geojson_simplifier/plotter.py]: Moved.

**Pull request merged**